### DISCUSSION TITLE
Whisper model large outputs affected by Dataset bias

### DISCUSSION THREAD

**Discussion by fluen-ai** [deepgram/community#357] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

Just letting you know, since I'm encountering a lot of these dataset bias hallucinations when making deepgram requests using the whisper-large model. Basically, there's often additional text being returned that isn't present in the audio. 
The problem is well explained here:
https://github.com/openai/whisper/discussions/928

I'm still hesitant to switch my stack to nova as it looked like it was still struggling to recognize product and people's names, and overall it seemed to work better for my use case (at least it was when I did my tests ~6mo ago). I'll give nova-2 a try when I get a chance, but just wanted to flag this problem in case Deepgram wants to address it (there are some suggestions in the GH link above). Thank you.

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?callback=<redacted>&detect_language=true&model=whisper-large&diarize=true&punctuate=true&paragraphs=true&utterances=true&numerals=true&measurements=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

9f0560cf-ef80-48c5-93b3-5d3e468652eb

**Comment by lukeocodes** [comment, marked as answer]
There is a good chance `nova-2-ea` will represent a big improvement around entities. I recommend you try it out. We also have a separate improvement for entity detection coming very soon.

Unfortunately, we use essentially what is the open source whisper model. If the model is hallucinating, it may require OpenAI to release an upstream fix before we can adopt it here - if we can.
